db/dataGen.js

```diff
@@ -42,46 +42,73 @@ const itemsPerProduct = () => {
 
   for (var i = 0; i <= 100; i += 1) {
     let item = {};
-    item.id = i;
+    item._id = i;
     item.reviews = reviewsPerItem();
     itemsArr.push(item);
   }
   return itemsArr;
 };
-//id: 1, id: 20 => id: 120
-const products = () => {
-  const products = [];
+
+const items = () => {
```

Comment:
I would like you to break the items function into three very clear parts that work independently.  One that makes pants, shoes, and shirts.  Right now it takes too much work to understand what this function is doing.  Add a comment for each of the parts so it is clear what it does.